docs/tutorial/6_visualization.md

```diff
@@ -59,84 +58,171 @@ This takes a list which defines which axes to plot as the slice and which to use
 
-```{code-cell} ipython3
-:tags: [skip-execution]
-
-ds[0, :, 400, :].plot()
+```{code-cell} ipython
+# Plot the same data as above
+ds[0, 0, :, 0].plot()
 plt.show()
 ```
 
-## More advanced plotting
+### Using AIA as a Context Image
+
+#### Fetching an AIA Image
+Next we are going to use the coordinate information in the VISP dataset to plot VISP's field of view over an AIA image.
+To do this we are going to use {obj}`sunpy.map`.
 
-For the next few examples we'll go back to using some VBI data.
+```{code-cell} python
+import sunpy.map
+from sunpy.net import Fido, attrs as a
+import astropy.units as u
+```
 
-```{code-cell} ipython3
-res = Fido.search(a.dkist.Dataset("AJQWW"))
-asdf_file = Fido.fetch(res, path="~/dkist_data/{instrument}_{dataset_id}")
+First we shall search for an AIA image closest to the start time of the VISP dataset.
 
-# We extract the top left tile of the VBI mosaic
-ds = dkist.load_dataset(asdf_file)[0, 0]
+```{code-cell} python
+start_time = ds.inventory["startTime"]
+end_time = ds.inventory["endTime"]
 ```
 
-Now let's take a slice of the data and plot it.
-This returns an axes object which we haven't needed before, but this time we'll assign it to a variable so that we can manipulate the plot.
-This allows us to do a number of things with it, such as show the grid of the plot.
+```{code-cell} python
+res = Fido.search(a.Time(start_time, end_time, start_time), a.Instrument.aia, a.Wavelength(19.3 * u.nm)) # change this to 160?
+res
+```
 
-```{code-cell} ipython3
-ax = ds[0].plot()
-ax.grid(True)
+```{code-cell} python
+aia_files = Fido.fetch(res)
+aia_files
 ```
 
-What you will probably notice here is that the grid is not aligned with the pixel grid of the image.
-This is because the plot returns a `WCSAxesSubplot`, which is coordinate-aware and automatically uses the world coordinate system for the grid.
-It also supports all the usual ways of manipulating subplots.
+Now we have an AIA image file, let's read it with `sunpy`.
 
-Since the `WCSAxesSubplot` is coordinate-aware, we can also use it for plotting coordinates directly, without having to do any manual conversions.
-To do this, we can use the `.plot_coord()` method.
+```{code-cell} python
+import sunpy.map
+```
 
-```{code-cell} ipython3
-import astropy.units as u
-from astropy.coordinates import SkyCoord
+```{code-cell} python
+aia = sunpy.map.Map(aia_files)
+aia
+```
 
-coord = SkyCoord(-181*u.arcsec, 112*u.arcsec, frame='helioprojective', observer='earth', obstime=ds.headers['DATE-AVG'][0])
+Now let's make a simple plot of the AIA map.
+The sunpy map object has a `.plot()` method which sets up a lot of stuff for us, but we will manually make the figure and axes.
 
-ax = ds[0].plot()
-ax.grid(True)
-# Plot the coordinate as a white circle
-ax.plot_coord(coord, 'wo')
-plt.show()
+```{code-cell} python
+fig = plt.figure()
+ax = plt.subplot(projection=aia)
+aia.plot(axes=ax)
+```
+
+#### Computing the VISP field of View
```

Comment:
These examples probably suffer from the issue that they wont handle rotation properly. I think the best thing to do would be to either leave this for now or use `sunpy.visualization.drawing.extent` from 6.1.